cypress/integration/search.spec.js

```diff
@@ -1,22 +1,20 @@
-describe.skip('Search', () => {
+describe('Search', () => {
   beforeEach(() => {
-    cy.visit('/');
+    cy.visit('/search');
```

Comment:
Skal `/search` ta over for `/` som "hjem" nå?

Reply:
nei, navigerer dit kun for å teste søk. Søkeboksen ligger kun der foreløpig, og ikke på `/`. Tenker vi kan vente med å putte en søkeboks der til søk faktisk fungerer

Reply:
åja, lurt!
hva tenker du på sikt? skal vi ha både `/` og `/search` hvor begge viser `Dashboard`?
Jeg tenkte først at vi bare trenger `/`, men da kan vi kanskje ikke ha søketermen i URLen (hvis vi vil ha det) 🤔

Reply:
jeg tenker at på forsiden så kan kun søkeboksen ligge, også når man faktisk gjør et søk, så navigeres man til `/search?query=test`

Reply:
men sidene er ellers helt like?
altså begge bruker `Dashboard` i `AppRoutes`?

Reply:
Nei, Dashboard er på `/`, Search er på `/search`

Reply:
Det er der jeg ikke ser forskjellen... skal ikke begge vise Searchbar og liste med publikasjoner? (tenker spesielt på kodeduplisering)

Reply:
men det skal jo være mye mer på forsiden enn på selve søkesiden? det er jo det samme som på google. forside: kun søkebar, når du søker: søkeside med resultat